xenit-alfresco/templates/solr/solr-stateful-set.yaml

```diff
@@ -29,64 +29,70 @@ spec:
-          successThreshold: {{ .Values.solr.readinessProbe.successThreshold }}
-          timeoutSeconds: {{ .Values.solr.readinessProbe.timeoutSeconds }}
+          readinessProbe:
+            failureThreshold: {{ .Values.solr.readinessProbe.failureThreshold }}
+            httpGet:
+              path: /solr/alfresco/xenit/actuators/readiness
+              port: 8080
+              scheme: HTTP
+            initialDelaySeconds: {{ .Values.solr.readinessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.solr.readinessProbe.periodSeconds }}
+            successThreshold: {{ .Values.solr.readinessProbe.successThreshold }}
+            timeoutSeconds: {{ .Values.solr.readinessProbe.timeoutSeconds }}
         {{- end }}
-        lifecycle:
-          postStart:
-            exec:
-              command:
-                - /bin/bash
-                - -c
-                - rm -f /opt/alfresco-search-services/data/index/archive/index/write.lock
-                - rm -f /opt/alfresco-search-services/data/index/alfresco/index/write.lock
-        envFrom:
-          - configMapRef:
-              name: solr-configmap
+          lifecycle:
+            postStart:
+              exec:
+                command:
+                  - /bin/bash
+                  - -c
+                  - rm -f /opt/alfresco-search-services/data/index/archive/index/write.lock
+                  - rm -f /opt/alfresco-search-services/data/index/alfresco/index/write.lock
+          envFrom:
+            - configMapRef:
+                name: solr-configmap
           {{- if .Values.solr.envFrom }}
           {{ toYaml .Values.solr.envFrom | nindent 10 }}
           {{- end }}
-        ports:
-          - containerPort: 8080
-            protocol: TCP
-          - containerPort: 8443
-            protocol: TCP
+          ports:
+            - containerPort: 8080
+              protocol: TCP
+            - containerPort: 8443
+              protocol: TCP
         {{- if or (.Values.solr.resources.requests) ((.Values.solr.resources.limits)) }}
-        resources:
+          resources:
           {{- if .Values.solr.resources.requests }}
-          requests:
+            requests:
           {{ toYaml .Values.solr.resources.requests | nindent 12 }}
           {{- end }}
           {{- if .Values.solr.resources.limits }}
-          limits:
+            limits:
           {{ toYaml .Values.solr.resources.limits | nindent 12 }}
           {{- end }}
         {{- end }}
-        volumeMounts:
-          - name: data
-            mountPath: /opt/alfresco-search-services/data
-            subPath: solr/data
+          volumeMounts:
+            - name: data
+              mountPath: /opt/alfresco-search-services/data/index
+              subPath: solr/data/index
+            - name: data
+              mountPath: /opt/alfresco-search-services/data/model
+              subPath: solr/data/model
+            - name: backup
```

Comment:
To avoid collisions, can we put solr6Backup in another path?
`/opt/alfresco-search-services/backup` ?